error-handling/src/main/java/com/palantir/remoting2/errors/SerializableStackTraceElement.java

```diff
@@ -30,13 +30,23 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public abstract class SerializableStackTraceElement implements Serializable {
 
-    public abstract Optional<String> getClassName();
+    public abstract String getClassName();
 
-    public abstract Optional<String> getMethodName();
+    public abstract String getMethodName();
 
     public abstract Optional<String> getFileName();
 
-    public abstract Optional<Integer> getLineNumber();
+    public abstract int getLineNumber();
+
+    @Override
+    public final String toString() {
+        StackTraceElement element = new StackTraceElement(getClassName(),
```

Comment:
cannot guarantee that these are non-null, so cannot simply construct this. IMO, it's reasonable to do:
```
StackTraceElement element = new StackTraceElement(
  getClassName().orElse("UnknownClass"),
  getMethodName().orElse("unknownMethod"),
  getFileName.orElse(null),
  getLineNumer.orElse(0));
```